File: VB/CalculatedAlias.Module/Updater.vb
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.BaseImpl

Namespace CalculatedAlias.Module
    Public Class Updater
        Inherits ModuleUpdater

        Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub
        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()

            Dim productOne As Product = ObjectSpace.CreateObject(Of Product)()
            productOne.Manufacturer = "Company A"
            productOne.Model = "Model One"
            productOne.Revision = "Revision One"
            productOne.Save()

            Dim productTwo As Product = ObjectSpace.CreateObject(Of Product)()
            productTwo.Manufacturer = "Company B"
            productTwo.Model = "Model Two"
            productTwo.Revision = "Revision Two"
            productTwo.Save()

            Dim productThree As Product = ObjectSpace.CreateObject(Of Product)()
            productThree.Manufacturer = "Company C"
            productThree.Model = "Model Three"
            productThree.Revision = "Revision Three"
            productThree.Save()
        End Sub
    End Class
End Namespace

File: CS/CalculatedAlias.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace CalculatedAlias.Module {
    public class Updater : ModuleUpdater {
        public Updater(Session session, Version currentDBVersion) : base(session, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();

            Product productOne = new Product(Session);
            productOne.Manufacturer = "Company A";
            productOne.Model = "Model One";
            productOne.Revision = "Revision One";
            productOne.Save();

            Product productTwo = new Product(Session);
            productTwo.Manufacturer = "Company B";
            productTwo.Model = "Model Two";
            productTwo.Revision = "Revision Two";
            productTwo.Save();
          
            Product productThree = new Product(Session);
            productThree.Manufacturer = "Company C";
            productThree.Model = "Model Three";
            productThree.Revision = "Revision Three";
            productThree.Save();     
        }
    }
}
